.buildkite/dagster-buildkite/dagster_buildkite/steps/trigger.py

```diff
@@ -28,6 +29,12 @@ def trigger_step(
         "trigger": pipeline,
         "label": f":link: {pipeline} from dagster@{commit}",
         "async": async_step,
+        "build": {
```

Comment:
Should these always get set or should we add an `env` argument instead? There's actually a decent amount of behavior inside this `trigger_step` that seems fairly specific to the needs of specific calls.